SCClassLibrary/Common/Core/Object.sc

```diff
@@ -137,6 +143,16 @@ Object  {
 		^this.performWithEnvir(selector, ().putPairs(pairs))
 	}
 
+	performWith {|selector, argumentsArray, keywordArgumentEnvir|
```

Comment:
As long as we don't have a primitive, this could be optimised. I think that performList is faster than the star, but I am not sure.

```
performWith {|selector, argumentsArray, keywordArgumentEnvir|
		var method, argList;
		if(keywordArgumentEnvir.isNil) { 
				^this.performList(selector, argumentsArray) 
		};
		method  = this.class.findRespondingMethodFor(selector) ?? {
			^this.doesNotUnderstand(selector)
		};
		argList = method.makePerformableArray(argumentsArray, keywordArgumentEnvir;
		^this.performList(selector, argList)
		)
	}
```